(DOCS-3898) Update log collection instructions and screenshots for ne… (#15066)
DataDog/documentation: content/en/logs/guide/send-aws-services-logs-with-the-datadog-lambda-function.md

2. [Enable logging](#enable-logging-for-your-aws-service) for your AWS service (most AWS services can log to a S3 bucket or CloudWatch Log Group).
3. [Set up the triggers](#set-up-triggers) that cause the Forwarder Lambda to execute when there are new logs to be forwarded. There are two ways to configure the triggers.

> **Note**: If you are in AWS `us-east-1` region, leverage [Datadog-AWS Private Link][2].
> 
> **Note**: Cloudformation creates an IAM policy which includes KMS:Decrypt for all resources. It is known that this does not align with Security hub best practice. The reason this permission is used is to decrypt objects from KMS encrypted S3 buckets to set up Lambda function, and which KMS key is used to encrypt the S3 buckets cannot be predicted. You can safely delete this permission after the installation successfully finished.

Reviewer: ```suggestion
If you are in the AWS `us-east-1` region, leverage [Datadog-AWS Private Link][2].

CloudFormation creates an IAM policy which includes KMS:Decrypt for all resources, and does not align with AWS Security Hub's best practice. This permission is used to decrypt objects from KMS-encrypted S3 buckets to set up Lambda function, and which KMS key is used to encrypt the S3 buckets cannot be predicted. You can safely delete this permission after the installation successfully finished.
```

Author: Thanks for these suggestions, I hadn't looked into this paragraph as it wasn't part of the update. I've added most of your suggestions but left the **Note:**, as this may be intentional to bring attention to this. Could you clarify the reason for removing the Note, so that I can follow up to confirm with the AWS team?

Reviewer 2: I think these changes are fine but we should leave the **Note:** unless there is a strong reason to remove

    | `logs:DeleteSubscriptionFilter`                             | Remove a Lambda trigger based on CloudWatch Log events                       |
    | `logs:DescribeSubscriptionFilters`                          | Lists the subscription filters for the specified log group.                  |

3. In the [AWS Integration page][4], select the AWS Account to collect logs from, and go to the **Log Collection** tab.  
   {{< img src="logs/aws/aws_log_setup_step1.png" alt="The Log Collection tab of the AWS integration page for a specific AWS account with instructions to send AWS Services logs and a textbox to autosubscribe the Forwarder Lambda function by entering the ARN of the Forwarder Lambda function" popup="true" style="width:80%;" >}}

> 4. Enter the ARN of the Lambda created in the previous section, and click **Add**.

Reviewer: ```suggestion
4. Enter the ARN of the Lambda created in the previous section and click **Add**.
```

5. Select the services from which you'd like to collect logs, and click **Save**. To stop collecting logs from a particular service, uncheck it.
   {{< img src="logs/aws/aws_log_setup_step2.png" alt="The Log Collection tab of the AWS integration page for a specific AWS account with one Lambda function successfully entered under Included ARNs and some of the services enabled under Log Sources" popup="true" style="width:80%;" >}}
6. If you have logs across multiple regions, you must create additional Lambda functions in those regions and enter them in this page.

> 7. To stop collecting all AWS logs, click the trash bin icon which appears when hovering over each Lambda. All triggers for that function are removed.

Reviewer: ```suggestion
7. To stop collecting all AWS logs, hover over a Lambda and click the Delete icon. All triggers for that function are removed.
```

> 8. Within a few minutes of this initial setup, your AWS Logs arrive in your Datadog [log explorer page][5] in near real-time.

Reviewer: ```suggestion
8. Within a few minutes of this initial setup, your AWS Logs appear in the [Log Explorer][5].
```
